spec/offline_sort/offline_sort_spec.rb

```diff
@@ -1,113 +1,128 @@
 require 'spec_helper'
+require 'byebug'
```

Comment:
I think we either need to drop this require or add the gem as a development dependency to https://github.com/panorama-ed/offline-sort/blob/master/offline-sort.gemspec so that folks will be guaranteed to have this available when they download the gem.

Reply:
Ack snuck in there thanks for catching, going to remove!